### DISCUSSION TITLE
"--json" versus "--output=json"

### DISCUSSION THREAD

**Discussion by hacktor** [AsamK/signal-cli#626] [Q&A] [answered]
Hi. I've just upgraded to v0.8.3 (was 0.7.2) and started getting this warning:

>WARN DaemonCommand - "--json" option has been deprecated, please use the global "--output=json" instead.

The command is started as "signal-cli  -u <phone> daemon --json". Where should I use this global --output= option? Obviously not by simply replacing --json with --output=json, as that is not recognized:

>signal-cli: error: unrecognized arguments: '--output=json'

**Comment by AsamK** [comment, marked as answer]
Global options go before the subcommand, in this daemon. 
`signal-cli -u Xxx --output=json daemon `

**Reply by hacktor** [reply]
thank you, this works.
And thank you for your work on this great project, it is greatly appreciated.
